src/pages/HealthContent.module.css

```diff
@@ -5,3 +5,9 @@
   padding-top: 40px;
   padding-bottom: 25px;
 }
+
+.overlay {
+  background-color: rgba(255, 255, 255, 0.6);
+  margin: -2.5em 0;
+  padding: 2.5em 0;
```

Comment:
wasn't there an opacity setting in here or something?

Reply:
this is the original code that I had. The opacity is in the background-color because I used rgba. But I lost this code when I was attempting to use git stash